docs/CTCORECHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CleverTap Android SDK CHANGE LOG
 
+### Version 4.0.4 (Mar 2, 2021)
+* Fixed FCM token refresh issue when multiple Firebase Projects are integrated in the application
```

Comment:
@kumaratulismu Add the following point - "If you're using multiple Firebase projects in your app, use this version instead of v4.0.0"

Reply:
@darshanclevertap updated the change log. please review once.